src/Layout/Auth/StyledAuth.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+	width: 100vw;
+	max-width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+`;
+
+export const Root = styled.div`
+	width: 50vw;
+	min-height: 50vh;
+	border: 1px solid #000;
```

Comment:
it's better to add all of the colors in one file like this

export const COLORS ={
black: #fff;
white: #000;
}